index.html

```diff
@@ -2,13 +2,11 @@
 <html lang="en">
 <head>
 
-<body>
 
-<a target="_blank"href="https://en.wikipedia.org/wiki/Nature">
-<video src="video1.mp4" width="500px" controls autoplay muted loop>
-  <source src="video1.mp4" type="video/mp4">
-  <source src="video1.webm" type="video/webm">
+<link rel="icon" type="image/jpg" href="favicon.jpg">
+<title>My website </title>
```

Comment:
The link and title should be inside the <head> tag

Reply:
Ohh okay I will